src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxDeployOperation.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.SqlServer.Dac;
+using Microsoft.SqlTools.ServiceLayer.DacFx.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.Utility;
+using System;
+using System.Data.SqlClient;
+using System.Diagnostics;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx
+{
+    /// <summary>
+    /// Class to represent an in-progress deploy operation
+    /// </summary>
+    class DacFxDeployOperation : DacFxOperation
```

Comment:
what's the thinking of how this relates to the existing Tasks support?

Reply:
Do you mean the TaskServices api?